selfdrive/manager/manager.py

```diff
@@ -56,7 +57,7 @@ def manager_init():
 
   # is this dashcam?
   if os.getenv("PASSIVE") is not None:
-    params.put_bool("Passive", bool(int(os.getenv("PASSIVE"))))
+    params.put_bool("Passive", bool(int(os.getenv("PASSIVE", "0"))))
```

Comment:
Does mypy need this? We already check that `PASSIVE` is set in the line above, and now it looks like this`"0"` is special but it will never actually be used.

Reply:
It does, otherwise it doesn't know the type of the os.getenv